tests/Flaps/Storage/DoctrineCacheAdapterTest.php

```diff
@@ -46,6 +46,15 @@ public function testValue()
     }
 
     /**
+     * @covers BehEh\Flaps\Storage\DoctrineCacheAdapter::incrementValue
+     */
+    public function testIncrementValue()
+    {
+        $this->setExpectedExceptionRegExp(\Exception::class, '/not implemented/');
```

Comment:
To ensure compatibility with PHP 5.3 and 5.4, we can't use the construct `Exception::class` here. (And it is the reason CI is currently failing for these versions).

Consider changing it to `'Exception'` or raising the required php version.